src/ScottPlot5/ScottPlot5/Primitives/SvgImage.cs

```diff
@@ -17,9 +20,20 @@ public SvgImage(int width, int height)
         Canvas = SKSvgCanvas.Create(rect, Stream);
     }
 
-    public string GetXml()
+    public string GetXml(bool isUtf = false)
     {
-        return Encoding.ASCII.GetString(Stream.ToArray()) + "</svg>";
+        Canvas.Flush();
+
+        if (isUtf)
```

Comment:
I think this might all be unnecessary, since the user isn't able to pass in ASCII, only unicode. Plus ASCII is forwards-compatible with utf8 as long as the top bit of each byte is zero (and if it isn't it's non-standard so there's not really a standard that we need to try and stick to).

I think we can just swap out ASCII for UTF8 across the board without needing any option to switch between them.

Reply:
updated PR to simplify the changes